src/OrbitBase/Tracing.cpp

```diff
@@ -200,9 +200,9 @@ void InitializeTracing() {
   g_orbit_api_v0.track_uint64 = &orbit_api_track_uint64;
   g_orbit_api_v0.track_float = &orbit_api_track_float;
   g_orbit_api_v0.track_double = &orbit_api_track_double;
-  g_orbit_api_v0.initialized = 1;
   std::atomic_thread_fence(std::memory_order_release);
-  g_orbit_api_v0.active = 1;
+  g_orbit_api_v0.initialized = 1;
+  g_orbit_api_v0.enabled = 1;
```

Comment:
Now that you have two assignments I'm not sure that all the reasoning around that fence still applies. Does it?

Reply:
I think so, yes. I added a paragraph in Orbit.cpp:

  // By the time we reach this, the "initialized" guard variable has been set to 1 and we know that
  // all function pointers have been written to and published to other cores by the use of
  // acquire/release fences. The "enabled" flag serves as a global toggle which is always used in
  // conjunction with the "initialized" flag to determine if the Api is active. See
  // orbit_api_active() in Orbit.h.

Reply:
Makes sense.